examples/tuple-destructure-binding-pattern/tuple_destructure_binding_pattern.bal

```diff
@@ -4,21 +4,21 @@ public function main() {
 
     // Tuple variables can be destructured with new values when used as a tuple variable
     // reference based binding pattern.
-    [string, [int, boolean]] [a1, [a2, a3]] = ["Hello", [123, true]];
+    [string, [int, boolean]][a1, [a2, a3]] = ["Hello", [123, true]];
```

Comment:
Here we need the space right?

Reply:
Yes. Thanks, Created the issue https://github.com/ballerina-platform/ballerina-lang/issues/21003